packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -433,7 +439,10 @@ const client =
                 // Must be named "client". See - https://www.npmjs.com/package/webpack-hot-server-middleware#usage
                 name: CLIENT,
                 // use source map to make debugging easier
-                devtool: mode === development ? 'source-map' : false,
+                devtool:
+                    mode === development || process.env.PWA_KIT_SOURCE_MAP === 'true'
```

Comment:
This line of code could evaluate to true under various circumstances, e.g. if it's in dev mode, or is the env var is set to true. As per the docs  [here](https://webpack.js.org/configuration/devtool/) "true" isn't a valid value for this sitting. So looking at the code it's hard to determine what webpack is going to do with that value, its possible that things just coincidentally work.

Reply:
@bendvc the value of it would either be 'source-map' or false, `mode === development || process.env.PWA_KIT_SOURCE_MAP === 'true'` is just the checking condition. The full statement is like this 
```
mode === development || process.env.PWA_KIT_SOURCE_MAP === 'true' ? 'source-map' : false

```

Reply:
oh.. apologies. I must have had tunnel vision because of the comments pushing the other line down.  👍